Makefile

```diff
@@ -8,4 +8,7 @@ maple:
 python-dynamic-module:
 	cd pythonx/clap && $(MAKE_CMD) build
 
-.PHONY: all maple python-dynamic-module
+install:
```

Comment:
I'm reluctant to use `install`, can we use `bin/maple` or something else?

Reply:
never mind, it was just a proposal.